src/antsibull/announcements.py

```diff
@@ -72,17 +67,17 @@
 )
 
 
-def email_heading(content):
+def forum_heading(content):
```

Comment:
We could potentially get rid of the `---------` style headings and just use standard markdown now that these are only being used on the forum. That can be a separate change.

Reply:
I agree; having this as a separate change is better I think. Probably some other improvements can be done to the announcement as well now that we can use proper MarkDown. (And `Register here to join the Ansible Forum:` can be removed :D)